src/components/ReserveForm.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import { createAppointment } from '../redux/Appointments/appointmentsSlice';
+
+const ReserveForm = () => {
```

Comment:
- [ ] When I try to reserve an investigator, this happens:

![image](https://user-images.githubusercontent.com/57421823/227432683-a81d5adf-e0ca-4f6d-84f9-954b4b3639cf.png)

This is because the app is claiming that I'm not logged in (tested with the default admin user).

If this is a bug, kindly fix it so your app works as intended. 👍